cachi2/core/package_managers/generic/models.py

```diff
@@ -35,3 +42,44 @@ class GenericLockfileV1(BaseModel):
 
     metadata: LockfileMetadata
     artifacts: list[LockfileArtifact]
+
+
+class GenericArtifact(BaseModel):
+    """Lockfile-version-independent representation of an artifact."""
+
+    download_url: str
+    target: Optional[str] = None
+    checksums: dict[str, str]
+    resolved_path: Optional[os.PathLike[str]] = None
```

Comment:
I'm getting an error when trying to run Cachi2 on a sample repo I have locally, I believe this line is the culprit:
```
raise PydanticSchemaGenerationError(
pydantic.errors.PydanticSchemaGenerationError: Unable to generate pydantic-core schema for os.PathLike[str].
 Set `arbitrary_types_allowed=True` in the model_config to ignore this error or implement `__get_pydantic_core_schema__` on your type to fully support it.
```

Reply:
Removed this class.